lib/Client/Transports/TransportHTTPPROTO.php

```diff
@@ -0,0 +1,59 @@
+<?php
+namespace LightStepBase\Client\Transports;
+
+class TransportHTTPPROTO {
+
+    protected $_host = '';
+    protected $_port = 0;
+    protected $_verbose = 0;
+
+    public function ensureConnection($options) {
+        $this->_verbose = $options['verbose'];
+
+        $this->_host = $options['collector_host'];
+        $this->_port = $options['collector_port'];
+
+        // The prefixed protocol is only needed for secure connections
+        if ($options['collector_secure'] == True) {
+            $this->_host = "ssl://" . $this->_host;
+        }
+    }
+
+    public function flushReport($auth, $report) {
+        if (is_null($auth) || is_null($report)) {
+            if ($this->_verbose > 0) {
+                error_log("Auth or report not set.");
+            }
+            return null;
+        }
+
+        $content = $report->toProto($auth)->serializeToString();
+
+        if ($this->_verbose >= 3) {
+            var_dump($content);
```

Comment:
Should we preface this dump with a log line indicating what it is a dump of?

Reply:
done